### DISCUSSION TITLE
Can we get Typescript @types support ?

### DISCUSSION THREAD

**Discussion by maxtsh** [GrapesJS/grapesjs#4043] [Ideas]
Hello and happy new year and thanks so much for this great package you've made.
I wanted to ask if we could have a typescript support type file for this package in near future ?

Thanks in advance.

**Comment by artf** [comment]
Hi @maxtsh yeah, in the long-term I'd like to add the full Typescript support but at the moment I don't have any ETA.

**Comment by Vac1911** [comment]
+1
